ddp-workspace/projects/ddp-singular/src/app/components/pages/for-clinicians/for-clinicians.component.ts

```diff
@@ -18,10 +18,10 @@ import { GTagEvent } from '../../../constants/gtag-event';
               can provide to you by mail.
             </p>
             <p>
-              Please contact us by email at
-              <a (click)="contactEmail($event)" href="mailto:contact@projectsingular.org">contact@projectsingular.org</a> or by phone at
-              <a (click)="contactPhone($event)" href="tel:+(650)7616486">(650) 561-6750</a> to let us know you are interested and the quantities of
-              each you would like to receive.
+              If you are interested in receiving these materials for your center or clinic, you can contact us by
+              email at <a (click)="contactEmail($event)" href="mailto:contact@projectsingular.org">contact@projectsingular.org</a>
+              or <a (click)="contactPhone($event)" href="tel:+(650)7616486">(650) 561-6750</a>. Please provide the
+              name of the recipient of these materials and their email address, phone number, and direct mailing address
```

Comment:
I believe that some of this text is meant to be bolded:

![Screen Shot 2022-12-08 at 10 29 36 AM](https://user-images.githubusercontent.com/29984380/206486951-132f715a-72af-45e4-8ef5-4895e25c7890.png)

Reply:
Maybe by the font-family will not be very noticeable: 
 
Here is an example with bold text:
<img width="737" alt="image" src="https://user-images.githubusercontent.com/118474256/206488583-5ea8c8ac-1103-48e1-a44f-d625ea4c7ab1.png">

Reply:
This is using <strong>, right?
Maybe using a different font for this part (I think they only care about appearance).
Try Spartan Bold?
https://www.cufonfonts.com/font/spartan

Reply:
Done! 

<img width="716" alt="image" src="https://user-images.githubusercontent.com/118474256/206498713-7d8e7f23-f0be-4d47-aa3f-5763973872aa.png">